### PR TITLE
rebuild labextension for short file names, test extension listing, pip check, add maintainer

### LINKED ISSUE #62: CondaVerificationError when installing latest ipyparallel on Windows
I'm not sure that is going on b/c I don't have a Windows machine to debug this now. I'll check later when I get my hands on a Windows laptop.

```
CondaVerificationError: The package for ipyparallel located at C:\Users\USER\Miniconda3\pkgs\ipyparallel-8.0.0-pyhd8ed1ab_0
appears to be corrupted. The path 'share/jupyter/labextensions/ipyparallel-labextension/static/vendors-node_modules_css-loader_dist_runtime_api_js-node_modules_css-loader_dist_runtime_cssW-313cda.e8f82e0a569f9c85299c.js.map'
specified in the package manifest cannot be found.
```

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.
  * [x] includes `third-party-licenses.json` from webpack build

Notes:
- fixes #62 
  - by rebuilding the js assets to have names short enough for windows
- reworks some metadata, jinja vars, etc.
- adds some metadata
- adds self as maintainer :wave:

### CODE REVIEW

**Comment by bollwyvl** [comment]
I think `ignore_run_exports` does the thing (it doesn't get installed during test). 

What I _don't_ know how to do is to signal that we _really_ don't care about the version of nodejs, we just need one build.

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
-
+  script:
+    - jlpm --ignore-scripts --ignore-optional
+    - jlpm clean:all
+    - jlpm build:prod
+    - {{ PYTHON }} -m pip install -vv --no-deps .
 
 requirements:
   build:
+    - nodejs ==14.*
```
Comment (jakirkham) [inline recipe/meta.yaml]: Maybe this would work for relaxing `nodejs` version?

```suggestion
    - nodejs *
```

**Comment by jakirkham** [comment]
Thanks Nick! 😄 This is great 🎉

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml]: sure, can't hurt

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
   url: https://pypi.io/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 95305a886f2c42e9603c034ea684e5c031d9d4222c66ed6d85eb3ae15d631e4b
 
 build:
-  number: 0
+  number: 1
   noarch: python
+  ignore_run_exports:
+    - nodejs
```
Comment (jakirkham) [inline recipe/meta.yaml]: After trying a bunch of stuff locally, seems we needed this also

```suggestion
    - nodejs *
```

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render

**Comment by jakirkham** [comment]
Have retested on Windows and it passes the build and package install now ( https://github.com/conda-forge/ipyparallel-feedstock/pull/63#issuecomment-984296023 )! 🚀

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@minrk if you have a moment, could you please take a look as well? 🙂

**Comment by minrk** [comment]
Wonderful to wake up to a bug report that's already fixed both here and upstream ❤️

**Comment by bollwyvl** [comment]
FOSS is like waffle house: the coffee is hot 24/7/365

File: recipe/meta.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ build:
   number: 1
   noarch: python
   ignore_run_exports:
-    - nodejs
+    - nodejs *
```
Comment (bollwyvl) [inline recipe/meta.yaml:15]: 🌠

**Comment by ocefpaf** [comment]
Thanks @jakirkham for the testing and @bollwyvl for the quick fix!

**Comment by jakirkham** [comment]
Just a note, filed issue ( https://github.com/conda-forge/ipyparallel-feedstock/issues/65 ) to track dropping the workarounds here on the next `ipyparallel` release (since upstream will have the fix then)

**Pull request merged**